server/src/server.js

```diff
@@ -1,23 +1,33 @@
 import express from 'express'
-
+import bodyParser from 'body-parser'
+import morgan from 'morgan'
 import { checkInt, checkBool } from './check'
 import { config } from './globals'
-import apiVersion from './api/version'
+import apiVersion from './routes/version'
+import apiRoute from './routes/routes'
+import { notFound, devErr } from './handlers'
 
 export function startServer() {
   const app = express()
 
   if (checkBool('dev', config.dev)) {
     console.log('Running as development server')
     app.use(express.static('../client/build'))
-    app.use(express.static('../misc'))
   }
 
+  app.use(bodyParser.json()) // support json encoded bodies
+  app.use(bodyParser.urlencoded({ extended: true })) // support encoded bodies
+  app.use(morgan('dev'))
   app.use(apiVersion)
 
+  app.use('/api', apiRoute)
+
+  app.use(notFound)
+  app.use(devErr)
```

Comment:
Use `config.dev` for conditionally selecting which error handler to use

Reply:
done